helpers/calendar/ratios_and_utilization.py

```diff
@@ -149,14 +149,14 @@ def map_results_to_ratio_class(self,
             )
         return result
 
-    def percentage_formater(portion, total):
+    def percentage_formater(self, portion, total):
         """ Calculates the percentage of the entered portion to the total and returns it
          :params
             - portion, total
         """
         try:
             percentage = (portion/total) * 100
-            return round(percentage, 1)
+            return percentage
```

Comment:
> I would suggest not to round off the percentage values as shown in the screenshots, let the front end team decide on the number of decimal places they'd like to render.

The rounding off has been removed but the snapshot has not been updated @Bonifase.
Let me update it now.